code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -511,6 +511,18 @@
 	hat_offset = 0
 	var/obj/item/t_scanner/adv_mining_scanner/cyborg/mining_scanner //built in memes.
 
+/obj/item/robot_module/miner/be_transformed_to(obj/item/robot_module/old_module)

```

Comment:
this could probably be put into the main robot module proc checking for if theres more than one sprites instead of copypasting everytime a new icon would get added

Reply:
That's actually what I am trying to figure out how to do. Chances are I would have to make a Datum for robot skins, and I cannot into datums.

Reply:
From what I saw, borg transform could use more cleanup than this. Leaving it for now and coming back to it is okay.

Reply:
Yah, the entire borg transform thing is basically a mess in general which hasn't had any large work done in a long time. Just small stuff added here and there now and then, which increased the mess.